django/contrib/postgres/fields/array.py

```diff
@@ -298,15 +304,18 @@ def __call__(self, *args, **kwargs):
 
 
 class SliceTransform(Transform):
-
     def __init__(self, start, end, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.start = start
         self.end = end
 
     def as_sql(self, compiler, connection):
         lhs, params = compiler.compile(self.lhs)
-        return '%s[%%s:%%s]' % lhs, params + [self.start, self.end]
+        # self.start is set to 1 if slice start is not provided.
```

Comment:
This comment doesn't seem to match any code in `SliceTransform`. Does it belong elsewhere?

Reply:
IMO it's valuable, because it explains why we can always use `self.start` without an extra check.